src/components/cylc/workflow/index.js

```diff
@@ -218,15 +218,7 @@ const MutationsWrapper = Vue.component('mutations-wrapper', {
   },
   template: `
     <div>
-      <component
-        is="VSkeletonLoader"
-        :ref="widgetId"
-        :loading="isLoading"
```

Comment:
This is always true if you have nothing in the UI but the mutation. With Graph and Tree views, the subscriptions when started can change the view's `isLoading` state.

But the mutation widget doesn't trigger any subscription.

So instead we simply add a Mutation view. The view has a loading state, and displays "Loading..." immediately when loaded. Furthermore, with work to add mutations in the right places, this view might be hidden or removed I guess.